include/wlr/types/wlr_xdg_shell.h

```diff
@@ -96,6 +96,7 @@ enum wlr_xdg_surface_role {
 	WLR_XDG_SURFACE_ROLE_NONE,
 	WLR_XDG_SURFACE_ROLE_TOPLEVEL,
 	WLR_XDG_SURFACE_ROLE_POPUP,
+	WLR_XDG_SURFACE_ROLE_INERT_POPUP,
```

Comment:
Not a fan of this. We can already check whether a surface is an inert popup: `xdg_surface->role == WLR_XDG_SURFACE_ROLE_NONE && surface->role == &xdg_popup_surface_role`.

Reply:
I understand and was thinking what would be an alternative. Thanks for the suggestion, I'll make an alternate version that way.